src/utils/updateTokenPrices.ts

```diff
@@ -25,79 +25,13 @@ interface CoinGeckoReponse {
     usd: number
   }
 }
-const otherTokens = {
```

Comment:
I think it would be better to move the changes to this file + App.ts to another PR, so we can revert just that if something ends up going wrong once the new price fetching logic is engaged.

Reply:
Token price fetching is working with the changes in this PR. What do you think would be a reason for something to go wrong?